source/command-logging-and-monitoring/command-logging-and-monitoring.rst

```diff
@@ -150,8 +150,9 @@ Security
 --------
 
 Some commands and replies will contain sensitive data and in order to not risk the leaking of this
-data to external sources or logs their commands AND replies MUST be redacted from the events. The
-value MUST be replaced with an empty BSON document. The list is as follows:
+data to external sources or logs their commands AND replies AND corresponding error messages/exceptions 
```

Comment:
the error messages/exceptions bit is new; see the comments on DRIVERS-2394 for some context. the security team recommends we redact these just in case, so I went ahead and added the requirement for log messages and have expectations for it in the tests.